modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -481,6 +481,9 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.raft.snapshot_attempt|label:new[Introduced in 2025.01] Total number of attempts to download Raft snapshots triggered. (counter)
 |<prefix>.cluster.raft.snapshot_success|label:new[Introduced in 2025.01] Total number of successfully downloaded Raft snapshots. (counter)
 |<prefix>.cluster.raft.snapshot_fail|label:new[Introduced in 2025.01] Total number of failed Raft snapshot download attempts. (counter)
+|<prefix>.cluster.raft.inbound_queue_offered|label:new[Introduced in 2025.02] Total number inbound messages offered to the queue. (counter)
```

Comment:
There is a grammar mistake. It should be "total number of".

Reply:
Thank you, good catch

Reply:
thank you, good catch